### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by smburdick**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
```
[the remark this answers is not in the document]
Comment (smburdick) [reply .github/ISSUE_TEMPLATE/bug_report.md:10]: Hash marks are preferred by me as well

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,38 +1,41 @@
...
-**To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+## Steps to reproduce the behavior:
+1.
+1.
+1.
```
Comment (nelimee) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]:
```suggestion
1.
2.
3.
```

**Review by nelimee — APPROVED** [review]
I had the idea that we could maybe split the issue template in 2: one targeted at the backend part, and another one targeted at the frontend part.
This would allow us to automatically assign tags to the issue, and maybe ask for more specialised information. 
Also, this is more work and I am not sure we need that separation now. It can come at a later stage.

**Comment by smburdick** [comment]
> I had the idea that we could maybe split the issue template in 2: one targeted at the backend part, and another one targeted at the frontend part.
> 
> This would allow us to automatically assign tags to the issue, and maybe ask for more specialised information. 
> 
> Also, this is more work and I am not sure we need that separation now. It can come at a later stage.

I think we can have a single one for now, since it should be fairly obvious from the context of the issue, assuming the ticket was filled out sufficiently. Will merge.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (smburdick) [reply .github/ISSUE_TEMPLATE/bug_report.md:16]: Markdown will render 1,1,1 as 1,2,3.

**Pull request merged**